### PR TITLE
Complete milestone 3 for math magicians project

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this project, I added the logic needed to make the Calculator component I developed in the previous project actually work. For this purpose I followed these steps:

1. Copy the files calculator.js and operate.js into a logic/ directory in your project.
2. Analyze the files calculator.js and operate.js, be sure you understand what they do.
3. Import the files in your Calculator component.
4. Implement the event handlers you need to use the math logic from calculator.js and operate.js in your React component.
5. Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
6. Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```
Comment (DeliceLydia) [inline src/logic/operate.js:17]: - [ ] When I try to divide a number with `0` am getting this error
e

Normally it has to return `error` in the screen kindly fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @tacettinkustu👏,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/operate.js (hunk 1 above)
Comment (tacettinkustu) [reply src/logic/operate.js]: I fixed that problem

Comment (DeliceLydia) [reply src/logic/operate.js]: Well resolved this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @tacettinkustu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**